osbs/build/build_request.py

```diff
@@ -867,7 +889,13 @@ def render(self, validate=True):
                            # add the path to the plugin's
                            # configuration. This is done elsewhere.
                            None):
-                          self.spec.registry_secrets.value})
+                          self.spec.registry_secrets.value,
+
+                          ('buildstep_plugins', 'orchestrate_build', 'osbs_client_config'):
```

Comment:
Alternatively, if it's just mounted on `/etc/osbs.conf` then there's no configuration needed. The current approach definitely provides more flexibility, but I'm not sure it's needed at all. Unless of course, mounting a secret that contains a file `osbs.conf` on `/etc` is problematic.